src/Type/MixedType.php

```diff
@@ -162,6 +162,15 @@ public function getValuesArray(): Type
 		return AccessoryArrayListType::intersectWith(new ArrayType(new IntegerType(), new MixedType($this->isExplicitMixed)));
 	}
 
+	public function fillKeysArray(Type $valueType): Type
+	{
+		if ($this->isArray()->no()) {
+			return new ErrorType();
```

Comment:
Maybe an invalid keytype should be handled as error here too?

Reply:
No, that is irrelevant I think because we don't know anything about the key = it is mixed